locales-da-DK.xml

```diff
@@ -199,8 +199,8 @@
 
     <!-- LONG ROLE FORMS -->
     <term name="director">
-      <single>director</single>
-      <multiple>directors</multiple>
+      <single>instruktør</single>
```

Comment:
OK, that's correct then, thanks. It just sounded very close to the English instructor and I wanted to make sure. (dirigenter would be conductor in English, so no ambiguity there.)